thrifty.fpvars.example

```diff
@@ -230,4 +230,11 @@ vpc_eips_if_unattached_enabled_actions  = ["skip", "release"]
 vpc_nat_gateways_if_unused_trigger_enabled  = false
 vpc_nat_gateways_if_unused_trigger_schedule = "15m"
 vpc_nat_gateways_if_unused_default_action   = "notify"
-vpc_nat_gateways_if_unused_enabled_actions  = ["skip", "delete"]
\ No newline at end of file
+vpc_nat_gateways_if_unused_enabled_actions  = ["skip", "delete"]
+
+# DynamoDB table stale data exceeding max age
+dynamodb_table_stale_data_max_days = 90
+dynamodb_table_stale_data_exceeding_max_age_trigger_enabled  = false
+dynamodb_table_stale_data_exceeding_max_age_trigger_schedule = "15m"
+dynamodb_table_stale_data_exceeding_max_age_default_action   = "delete_table"
```

Comment:
Value should be `"notify"`, allowing the user to override it but we shouldn't outright delete anything automatically out of the box.